src/main/java/soonkeatneo/duke/task/Deadline.java

```diff
@@ -0,0 +1,34 @@
+package soonkeatneo.duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Implementation for tasks with a specified end-date.
+ *  @author Soon Keat Neo
+ *  @version CS2103T AY20/21 Sem 2 iP
+ */
+
+public class Deadline extends Task {
+    private final LocalDate lastDate;
+    public Deadline(String taskName, String lastDate) {
+        super(taskName, "D");
+        this.lastDate = LocalDate.parse(lastDate);
+    }
+
+    /**
+     * Returns the date of the instance.
+     * @return date of the object
+     */
+    public LocalDate getDate() {
+        return this.lastDate;
+    }
+
```

Comment:
I believe the toString() method, even though overridden, needs a JavaDoc description still